doc/classes/Label.xml

```diff
@@ -129,14 +132,20 @@
 			[b]Note:[/b] If using a font with [member FontFile.multichannel_signed_distance_field] enabled, its [member FontFile.msdf_pixel_range] must be set to at least [i]twice[/i] the value of [theme_item outline_size] for outline rendering to look correct. Otherwise, the outline may appear to be cut off earlier than intended.
 			[b]Note:[/b] Using a value that is larger than half the font size is not recommended, as the font outline may fail to be fully closed in this case.
 		</theme_item>
+		<theme_item name="shadow_outline_size" data_type="constant" type="int" default="0">
+			The size of the shadow outline.
+		</theme_item>
 		<theme_item name="shadow_offset_x" data_type="constant" type="int" default="1">
 			The horizontal offset of the text's shadow.
 		</theme_item>
 		<theme_item name="shadow_offset_y" data_type="constant" type="int" default="1">
 			The vertical offset of the text's shadow.
 		</theme_item>
-		<theme_item name="shadow_outline_size" data_type="constant" type="int" default="1">
-			The size of the shadow outline.
+		<theme_item name="draw_shadow_before_outline" data_type="constant" type="int" default="0">
+			Changes the order in which the [code]shadow[/code] and [code]shadow_outline[/code] is draw. If the value is [code]1[/code], the [code]shadow[/code] and the [code]shadow_outline[/code] will be [b]between[/b] the [code]text_outline[/code] and the main [code]text[/code], otherwise the shadow will be in the background by default
```

Comment:
```suggestion
			Changes the order in which the font [code]shadow[/code] and [code]shadow_outline[/code] is drawn. If the value is [code]1[/code], the [code]shadow[/code] and the [code]shadow_outline[/code] will be [b]between[/b] the [code]text_outline[/code] and the main [code]text[/code], otherwise the shadow will be in the background by default.
```